src/fileiter.cpp

```diff
@@ -378,7 +396,11 @@ inline _fi_find_handle find_first_file(const char* wild,  _fi_find_data& data)
    if (::MultiByteToWideChar(CP_ACP, 0,  wild, wild_size,  wide_wild, wild_size + 1) == 0)
       return _fi_invalid_handle;
 
+# if BOOST_PLAT_WINDOWS_RUNTIME
+   return FindFirstFileExW(wide_wild, FindExInfoStandard, &data, FindExSearchNameMatch, NULL, 0);
```

Comment:
According to the MSDN library documentation, FindFirstFileExW is available in Windows XP and Server 2003, so maybe it can be used unconditionally (like in your filesystem pull request)?

And the boost/predef/platform.h include for the BOOST_PLAT_* macros seems to be missing, also in your filesystem and test pull requests.